projects/storefrontlib/src/lib/cms/page-layout/page-layout.service.ts

```diff
@@ -163,18 +176,28 @@ export class PageLayoutService {
     return slotConfig;
   }
 
-  private noConfigFound(template: string, section?: string) {
-    if (section) {
-      if (!this.warnLogMessages[section + ':' + template]) {
-        console.warn(
-          `no layout config found for section ${section} of template ${template}`
-        );
-        this.warnLogMessages[section + ':' + template] = true;
-      }
-    } else if (!this.warnLogMessages[template]) {
-      console.warn(`no layout config found for ${template}`);
-      this.warnLogMessages[template] = true;
+  private logMissingLayoutConfig(page: Page, section?: string): void {
+    // only render warnings during development
+    if (this.config.production) {
+      return null;
```

Comment:
I would just keep simple 'return;' without 'null'.

Reply:
thanks, this actually a mistake. Will fix.